components/shared/net/lib.rs

```diff
@@ -595,10 +596,25 @@ impl ResourceFetchTiming {
             return;
         }
         match attribute {
-            ResourceAttribute::DomainLookupStart => self.domain_lookup_start = precise_time_ns(),
+            ResourceAttribute::DomainLookupStart => {
+                self.domain_lookup_start = SystemTime::now()
+                    .duration_since(UNIX_EPOCH)
+                    .unwrap_or_default()
+                    .as_nanos() as u64
+            },
```

Comment:
Instead of repeating this code so many times, maybe just the current time before this match statement to simplify things? If it makes sense, perhaps store all these values as `Duration` as well?